index.html

```diff
@@ -20,12 +20,152 @@
+                    <li class="codekitlist">Codekit</li>
+                    <li class="codepenlist">Codepen</li>
+                    <li class="githublist">GitHub</li>
+                    <li class="gitlablist">GitLAb</li>
+                    <li class="termlist">Terminal</li>
+                </ul>
+            </div>
+            <img class="newimg_grid33" src="Group_84.svg" alt="gridimage">
+            <img class="newimg1_grid33" src="Shape_grid33.svg" alt="gridimage">
+            <a name="abouttext">
+                <div class="abouttext">About</div>
+            </a>
+            <div class="aboutpara">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</div>
+            <div class="rectangle14_grid33">
+                <div class="getmycv">Get my CV</div>
+                <img class="img4" src="ic_download.svg" alt="download icon">
+            </div>
+        </section>
+        <section class="contactform_end">
+            <img class="img11" src="left_topside_icon.svg" alt="Image geometry on the left top side">
+            <img class="img112_getstarted" src="Shape1_getstarted.svg" alt="gridimage">
+            <img class="img111_getstarted" src="Shape_getstarted.svg" alt="gridimage">
+            <form name="Contactform_grid" action="https://formspree.io/f/xbjqawyz" onsubmit="return validateForm()" method="post" aria-required>
+                <a name="text10">
+                    <h1 class="text10">Get started</h1>
+                </a>
+                <p class="contact-form-para">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                <div class="namelabel">Your name</div>
+                <div class="emaillabel">Your email</div>
+                <div class="messagelabel">Message</div>
+                <ul class="contact-form-ullist">
+                    <li class="contact-form-name">
+                        <input type="text" maxlength="30" class="nooutline" id="name_grid" name="user_name" placeholder="Your name">
+                    </li>
+                    <li class="contact-form-email"><input type="email" id="mail_grid" class="nooutline" name="user_email" placeholder="Your email"></li>
+                    <li class="contact-form-message"><textarea maxlength="500" id="msg_grid" class="nooutline" name="user_message" placeholder="Enter your message here..."></textarea></li>
+                </ul>
+            </form>
+                <img class="getstarted_image" src="image-geometry_7.svg" alt="Gridimage">
+                <div class="footer_rect"></div>
+                <div class="text17_footer">Get my Resume</div>
```

Comment:
Get my Resume looks just like the design. :clap:

- [ ] Make sure this is a button.,

Reply:
I have made this as button.